pywbemtools/pywbemcli/_cmd_class.py

```diff
@@ -186,8 +186,13 @@ def class_get(context, classname, **options):
                      options_metavar=CMD_OPTS_TXT)
 @click.argument('classname', type=str, metavar='CLASSNAME', required=True,)
 @click.option('-f', '--force', is_flag=True, default=False,
+              help=u'Same as --include-instances. The -f / --force option has '
+                   'been deprecated and will be removed in a future version.')
+@click.option('--include-instances', is_flag=True, default=False,
               help=u'Delete any instances of the class as well. '
-                   'Some servers may still reject the class deletion. '
+                   'WARNING: Deletion of instances will cause all implemented '
```

Comment:
I do not understand the all in "cause all implemented". Might be better with "Deletion of instances can cause side-effects such as ..."

Reply:
I wanted to express that these effects are intentionally implemented in the server and even mandated in the mgmt profile if that was followed. I did not do a good job on that, though and should have gone much further.

That these "side-effects" happen is really the sole purpose of the CIM instrumentation. Calling these intended effects "side-effects" is really strange. This still comes from the notion that we are dealing with a database that "may have" some "side-effects". No, they are the main effects and totally intentional.

Feel free to propose a sentence that expresses that. And that ideally does not contain "side-effects" or "may". ;-)

Reply:
I have changed the text to now say:

```
    WARNING: Deletion of instances will cause the removal of corresponding
    resources in the managed environment (i.e. in the real world).
```
**DISCUSSION**

Reply:
How about changing will to can.  Will means absolutely this will happen and "depending on the CIM Instrumentation".   I hesitate to say will be cause there may be whole namespaces that do not deal with real instrumentation and which do not cause any failures and others for which deleting instances is a total disaster to the managed environment.

Reply:
Agreed that the text above is ok, but that more text should be added that clarifies that some classes do not even allow instance deletion.

Reply:
Add the sentence:
```
Some instances may not be deletable.
```
Also, added the warning about deletion of instances to the heklp text of the "instance delete" command.